openwisp_users/multitenancy.py

```diff
@@ -134,3 +135,47 @@ class MultitenantRelatedOrgFilter(MultitenantOrgFilter):
     """
 
     multitenant_lookup = 'organization__in'
+
+
+class BaseOrganizationPermission(object):
```

Comment:
should not extend object but a base permission class in DRF, please read https://www.django-rest-framework.org/api-guide/permissions/#custom-permissions, you then have to define `has_object_permission`.

You'll get the organization from the object passed to `has_object_permission`, but we have to take into account the case in which the organization field is not on the object itself but on a parent, we need to define a way to support this.